shared/ssa/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 0.0.1
+
+No user-facing changes.
```

Comment:
Maybe this should include a proper description of what this pack is?

> Extracted shared SSA QL code, used by all core language packs, into its own library pack.

Or something like that.

Reply:
+1

Reply:
Fixed.